app/controlplane/extensions/core/template/v1/extension.go

```diff
@@ -29,6 +29,7 @@ type Integration struct {
 	*sdk.FanOutIntegration
 }
 
+// 1 - API schema definitions
```

Comment:
I made some improvements in the template based on my experience using it now as base.